src/Core/Manifest/ModuleManifest.php

```diff
@@ -271,7 +271,7 @@ public function getModuleByPath($path)
         $modules = ModuleLoader::inst()->getManifest()->getModules();
         foreach ($modules as $module) {
             // Check if path is in module
-            if (stripos($path, realpath($module->getPath())) !== 0) {
+            if (stripos($path, (string)realpath($module->getPath())) !== 0) {
```

Comment:
```suggestion
            $realPath = realpath($module->getPath());
            if ($realPath && stripos($path, $realPath)) !== 0) {
```

I'd prefer something like this